reaktive/src/commonTest/kotlin/com/badoo/reaktive/completable/RetryTest.kt

```diff
@@ -79,7 +79,7 @@ class RetryTest : CompletableToCompletableTests by CompletableToCompletableTests
 
     @Test
     fun predicate_receives_valid_counter_WHEN_upstream_produces_error() {
-        val timeRef = AtomicInt()
+        val timeRef = AtomicLong(0)
```

Comment:
What kind of warning are you getting here?

Reply:
Fixed